### PR TITLE
Update azure_rm_securitygroup related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_securitygroup related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_securitygroup/azure_rm_securitygroup_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_securitygroup_facts.py [hunk 1]
```diff
@@ -60,21 +60,47 @@
...
-        "location": "eastus2",
-        "name": "secgroup001",
-        "properties": {
-            "defaultSecurityRules": [
-                {
-                    "etag": 'W/"d036f4d7-d977-429a-a8c6-879bc2523399"',
-                    "id": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroup/myResourceGroup/providers/Microsoft.Network/networkSecurityGroups/secgroup001/defaultSecurityRules/AllowVnetInBound",
-                    "name": "AllowVnetInBound",
-                    "properties": {
+    type: complex
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_securitygroup_facts.py:66]: type list

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_securitygroup_facts.py:66]: if describe nested return values, set type: complex and repeat the elements above for each sub-field (Azure format [document's link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html)).

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @Fred-sun for the Ansible documentation fix.

**Pull request merged**